src/types/models.ts

```diff
@@ -356,12 +364,16 @@ export interface UserIdentifier {
   email: string
 }
 
+export interface ConnectedAccountError extends SeamError {
+  is_connected_account_error: true
```

Comment:
is the intention to expose these booleans on Connect?

Reply:
Yes, because the list is flat.